client/src/components/Body/style.module.scss

```diff
@@ -0,0 +1,25 @@
+.gridContainer {
+    height: 100%;
+    padding: 30px;
+    background-color: #f0f2f5;
+}
+
+.noWidgetContainer {
+    margin: 0;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    margin-right: -50%;
+    transform: translate(-50%, -50%);
+    text-align: center;
+
+    .message {
```

Comment:
nit: fix tabbing

Reply:
This is intentional since the element with the message class is within the no widget container class so

Reply:
gotcha